farm/modeling/wordembedding_utils.py

```diff
@@ -22,16 +22,30 @@
 # the dicts need to be used with HF transformers to use their data + modelling functionality
 # language model config
 PRETRAINED_CONFIG_ARCHIVE_MAP = {
-    "glove-german-uncased": "https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-models/0.4.1/glove-german-uncased/language_model_config.json"}
+    "glove-german-uncased": "https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-models/0.4.1/glove-german-uncased/language_model_config.json",
+    "glove-english-uncased-6B": "https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-models/0.4.1/glove-english-uncased-6B/language_model_config.txt",
+    "glove-english-cased-840B": "https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-models/0.4.1/glove-english-cased-840B/language_model_config.txt",
+}
 # tokenization
 EMBEDDING_VOCAB_FILES_MAP = {}
 EMBEDDING_VOCAB_FILES_MAP["vocab_file"] = {
-    "glove-german-uncased": "https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-models/0.4.1/glove-german-uncased/vocab.txt"}
-MAX_MODEL_INPU_SIZES = {"glove-german-uncased": 10000}
-PRETRAINED_INIT_CONFIGURATION = {"glove-german-uncased": {"do_lower_case": False}}
```

Comment:
@Timoeller any special reason why `do_lower_case` was False for this uncased model or just a bug?

Reply:
it was a bug